<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWindow</class>
 <widget class="QWidget" name="SettingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 177, 149);</string>
  </property>
  <widget class="QPushButton" name="back_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(142, 197, 189,255);
border:2px outset rgb(255, 255, 255);
border-radius:8px;
text-align: center center; 
</string>
   </property>
   <property name="text">
    <string>返回主页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>241</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 204, 204,255);
border:2px outset rgb(255, 255, 255);
border-radius:8px;
text-align: center center; 
</string>
   </property>
   <property name="text">
    <string>自定义清扫设置</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>180</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 204, 204,255);
border-radius:8px;
text-align: center center; </string>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>光照上限(lux)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="illu_max">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>光照下线(lux)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="illu_min">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 204, 204,255);
border-radius:8px;
text-align: center center; </string>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>湿度上限(%)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="humi_max">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="humi_min">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>湿度下限(%)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>251</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 204, 204,255);
border-radius:8px;
text-align: center center; </string>
   </property>
   <layout class="QFormLayout" name="formLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>温度上限(°C)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="temp_max">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="temp_min">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>温度下限(°C)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>260</y>
     <width>251</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 204, 204,255);
border-radius:8px;
text-align: center center; </string>
   </property>
   <layout class="QFormLayout" name="formLayout_4">
    <item row="0" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>开始时间</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>结束时间</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QDateTimeEdit" name="endTimeEdit">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2025</year>
        <month>3</month>
        <day>27</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2026</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2025</year>
        <month>3</month>
        <day>27</day>
       </datetime>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="startTimeEdit">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2025</year>
        <month>3</month>
        <day>27</day>
       </datetime>
      </property>
      <property name="maximumDateTime">
       <datetime>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
        <year>2026</year>
        <month>12</month>
        <day>31</day>
       </datetime>
      </property>
      <property name="minimumDateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2025</year>
        <month>3</month>
        <day>27</day>
       </datetime>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>390</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>注意：所有参数上下限只需填写数值即可</string>
   </property>
  </widget>
  <widget class="QPushButton" name="apply_btn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>440</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(142, 197, 189,255);
border:2px outset rgb(255, 255, 255);
border-radius:8px;
text-align: center center; </string>
   </property>
   <property name="text">
    <string>一键设置</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
